File: playground/src/main/java/com/jiamian/translation/entity/dto/ModelVersionsApiDTO.java
package com.jiamian.translation.entity.dto;

import com.google.common.collect.Lists;
import lombok.Data;

import java.time.LocalDateTime;
import java.util.List;

/**
 * @author DingGuangHui
 * @date 2023/3/28
 */
@Data
public class ModelVersionsApiDTO {

	private long id = 0;
	private long modelId;
	private String name;
	private LocalDateTime createAt;
	private LocalDateTime updatedAt ;
	private List<String> trainedWords = Lists.newArrayList();
	private String baseModel;
	private int earlyAccessTimeFrame = 0;
	private String description = "";
	private String downloadUrl = "";
	private List<FilesApiDTO> files;
	private List<ImagesApiDTO> images;

}

File: web/src/main/java/com/jiamian/translation/entity/dto/api/ModelVersionsApiDTO.kt
package com.jiamian.translation.entity.dto.api

import com.google.common.collect.Lists
import lombok.Data
import java.time.LocalDateTime

/**
 * @author DingGuangHui
 * @date 2023/3/28
 */
@Data
class ModelVersionsApiDTO {
    var id: Long = 0
    var modelId: Long = 0
    var name: String? = null
    var createAt: LocalDateTime? = null
    var updatedAt: LocalDateTime? = null
    var trainedWords: List<String> = Lists.newArrayList()
    var baseModel: String? = null
    var earlyAccessTimeFrame = 0
    var description = ""
    var downloadUrl = ""
    var files: List<FilesApiDTO>? = null
    var images: List<ImagesApiDTO>? = null
}